src/main/resources/resources.properties

```diff
@@ -114,4 +115,4 @@ ui.authenticationView.tokenLink=What is my token?
 ### Misc
 ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin) in<br>programming courses that rely on the A+ learning management system, which has been<br>developed at Aalto University. The plugin accesses programming assignments<br>and automated grading services provided by A+ and otherwise enhances<br>the student experience.</html>
 ui.aboutDialog.title=About the A+ Courses Plugin
-ui.aboutDialog.website=A+ Courses Plugin website
+ui.aboutDialog.website=A+ courses plugin website
```

Comment:
I'd capitalize "Courses". "A+ Courses" is the name of the plugin and it should follow title capitalization, or am I wrong?